src/components/tasks/TasksContent.tsx

```diff
@@ -12,10 +12,18 @@ import { TasksLoader } from './TasksLoader';
 import PaginationButton from '../PaginationButton';
 import { useRouter } from 'next/dist/client/router';
 
+type routerQueryParams = {
+    dev?: string | false;
+    selected?: Tab;
+};
+
 export const TasksContent = () => {
     const router = useRouter();
-    const { dev = false } = router.query;
-    const [activeTab, setActiveTab] = useState(Tab.IN_PROGRESS);
+    const { dev = false, selected = Tab.IN_PROGRESS }: routerQueryParams =
```

Comment:
- please do not assign default values to the value we get from router.query, it is only meant to get the query params value from the url.

Reply:
fixed.